qiskit_ibm_runtime/api/rest/runtime.py

```diff
@@ -185,6 +191,31 @@ def logout(self) -> None:
         url = self.get_url("logout")
         self.session.post(url)
 
+    # IBM Cloud only functions
```

Comment:
Why do you use separate files for backend related adapters (`/rest/backend.py` vs. `/rest/cloud_backend.py`) and a single file for runtime related adapters?

Reply:
3 reasons:
1. `rest/backend.py` connects to API provided by the old API service (not the runtime service) vs `/rest/cloud_backend.py` connects to the `/devices` endpoints in the runtime service deployed on IBM Cloud. So they both connect to entirely different APIs offered by two different API services. Whereas the runtime related adapters connect to the same runtime API service, just hosted on two different platforms IQX vs IBM Cloud.
2. Since this is not a public facing class I thought it'd be clearer for the team to have separate files to connect to two distinct API services, the JSON responses are also slightly different.
3. We also have a new `/devices/{id}/configuration` endpoint in the runtime service on IBM Cloud which is not available in the old API service.

I am happy to put it all in one class if you all think it'd be easy to maintain and easily understandable for the team. @jyu00 @kt474 Thoughts?

Reply:
Thanks for the details. That convinced me for my part that using separate backend adapters (`/rest/backend.py`, `/rest/cloud_backend.py`) is a good choice. I'm also fine with sticking to a single file for runtime related adapters.